sdcflows/workflows/base.py

```diff
@@ -105,22 +105,29 @@ def init_sdc_estimate_wf(fmaps, epi_meta, omp_nthreads=1, debug=False):
         return workflow
 
     workflow.__postdesc__ = """\
-Based on the estimated susceptibility distortion, a corrected
-EPI (echo-planar imaging) reference was calculated for a more
-accurate co-registration with the anatomical reference.
+Based on the estimated susceptibility distortion, a corrected EPI (echo-planar imaging) reference
```

Comment:
Please rebase or merge with current upstream so differences in this file are real.

Reply:
As you can see I've opened a new PR (#106 based on the current master). Will respond to your comments here, but the perhaps better to continue the discussion in the new PR? Sorry for the mess, far from fluent in git...